modular_citadel/code/modules/client/loadout/_medical.dm

```diff
@@ -3,3 +3,21 @@
 	category = SLOT_NECK
 	path = /obj/item/clothing/neck/stethoscope
 	restricted_roles = list("Medical Doctor", "Chief Medical Officer")
+
+/datum/gear/bluescrubs
+	name = "Blue Scrubs"
+	category = SLOT_W_UNIFORM
+	path = /obj/item/clothing/under/rank/medical/blue
+	restricted_roles = list("Medical Doctor", "Chief Medical Officer", "Geneticist", "Chemist", "Virologist")
```

Comment:
yey.

Lastly you can use what I did for Star Trek uniforms by adding 

```
restricted_desc = "Medical"
```

So it only shows "medical" instead of all five roles

Reply:
Didn't know that was a thing, good to know. Thanks poojy.